relayer/ops/charts/hardhat/values.yaml

```diff
@@ -8,7 +8,7 @@ service:
 resources:
   limits:
     cpu: 5000m
-    memory: 5120Mi
+    memory: 8000Mi
```

Comment:
Hmm is this necessary? The current values for all the pods are so high that my 16GB RAM laptop refused to spawn 5 core nodes (last one was Pending, waiting for more system resources to free up). I've had to modify the settings locally to only spawn 3 nodes.

Reply:
i'll drop it back down :) i was testing something to see if I could make devnet more reliable